src/ElvSpace.js

```diff
@@ -106,6 +106,13 @@ class ElvSpace {
       });
       elvAccount.SetAccountTenantContractAddress(tenant.address);
 
+      // We must sign in using the new account to set the groups' metadata - metadata can only be set by the groups' admin
+      let account_wallet = this.client.GenerateWallet();
+      let account_signer = account_wallet.AddAccount({
+        privateKey: account.privateKey,
+      });
+      this.client.SetSigner({ signer: account_signer });
+
```

Comment:
Need to change account or else OnlyAdmin() on meta_object contract will block the following putMeta(s).